src/sentry/incidents/endpoints/serializers/alert_rule.py

```diff
@@ -66,6 +68,9 @@ class AlertRuleSerializerResponseOptional(TypedDict, total=False):
         "totalThisWeek",
         "latestIncident",
         "description",  # TODO: remove this once the feature has been released to add to the public docs, being sure to denote it will only display in Slack notifications
+        "sensitivity",  # The following three fields are for anomaly detection, which is behind a feature flag
```

Comment:
Lets not assume that these will stay in order here.
Best to add the note to each item if necessary